[Symfony] Process error with Internationalized routing

<!-- First, thank you for reporting a bug. That takes time and we appreciate that! -->

| Subject        | Details                                                      |
| :------------- | :----------------------------------------------------------- |
| Rector version | v0.6.6          |
| PHP version    |  PHP 7.3.2                            |
| Full Command   | vendor/bin/rector process bin src --set symfony41 --dry-run  | 

## Current Behaviour 

```
 [ERROR] Could not process "/var/www/symfony/src/Controller/ContactController.php" file, due to:
         "Argument 1 passed to Rector\BetterPhpDocParser\PhpDocNode\Symfony\SymfonyRouteTagValueNode::__construct() must
         be of the type string, null given, called in
         /var/www/symfony/vendor/rector/rector/packages/BetterPhpDocParser/src/PhpDocNodeFactory/Symfony/SymfonyRoutePhp
         DocNodeFactory.php on line 40".
```

## Minimal PHP Code Causing Issue

```php
class ContactController
{
    /**
     * @Route({
     *     "en": "/send-us-an-email",
     *     "nl": "/stuur-ons-een-email"
     * }, name="contact")
     */
    public function send()
    {
        // ...
    }
}
```

It looks like the problem is, that the constructor of the `SymfonyRouteTagValueNode` expects a string for the $path, but actually it can be an array.

--- a/packages/BetterPhpDocParser/src/PhpDocNode/Symfony/SymfonyRouteTagValueNode.php
+++ b/packages/BetterPhpDocParser/src/PhpDocNode/Symfony/SymfonyRouteTagValueNode.php
@@ -26,7 +26,7 @@ final class SymfonyRouteTagValueNode extends AbstractTagValueNode
     public $name;
 
     /**
-     * @var string
+     * @var string|null
      */
     private $path;
 
@@ -62,7 +62,7 @@ final class SymfonyRouteTagValueNode extends AbstractTagValueNode
      * @param string[] $requirements
      */
     public function __construct(
-        string $path,
+        ?string $path,
         ?string $name = null,
         array $methods = [],
         array $options = [],

